app/src/main/java/ro/stery/loginapplication/model/Repository.java

```diff
@@ -0,0 +1,136 @@
+package ro.stery.loginapplication.model;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Created by Stery on 04.04.2017.
+ */
+
+public class Repository {
+
+    String mName;
+    String mOwner;
+    String mDescription;
+    boolean mIsPrivate;
+    String mDefaultBranch;
+    List<String> mTopics;
+    int mWatchersCount;
+    Date mCreatedAt;
+    Date mUpdatedAt;
+
+    public void setmName(String mName) {
```

Comment:
Poti specifica in Android Studio (Preferences -> Code Style -> Java):
Fields sa aiba prefixul "m" si static fields sa aiba prefixul "s" -> Apply

Astfel cand iti va genera constructor + getters/setters sa tina cont de prefix si sa nu ajungi cu prefixul si in numele argumentelor sau al getterilor/setterilor

vezi urmatorul link: https://i.stack.imgur.com/g55ba.png